### ISSUE TITLE
Attempting to Run on 64 bit Pixel Device

### ISSUE THREAD

**Issue by ririvas** [lufinkey/react-native-spotify#126]
Just throwing this out there in case someone has a clue or advice. 

I'm trying to run the starter app I built on a 64 bit Pixel device. I've successfully run on 32 and 64 bit emulators. 

The application crashes when I run the RNSpotifyModule.java :: login method. Specifically, when debugging, at initializePlayerIfNeeded at line 391. 

I'm unable catch any errors or any useful information.

**Comment by lufinkey** [comment]
If you could get any sort of error message, that would be really helpful. Currently I am unable to reproduce this issue

**Comment by ririvas** [comment]
Thanks for looking into this. I did not specify the issue enough. Rather than a 32 vs 64 bit issue. The problem lay with the instruction set type. My device CPU is an arm64-v8a instruction set type. The emulators I ran on where x86/x86_64 instruction sets. 

I first tried running on emulator but arm emulators are terribly slow. So, on a whim, I modified the section below. Note, the react-native-spotify android install instructions specify to add the following. 

```
packagingOptions 
{
    pickFirst 'lib/armeabi-v7a/libgnustl_shared.so'
    pickFirst 'lib/x86/libgnustl_shared.so'
    exclude 'lib/arm64-v8a/libgnustl_shared.so' // this line removed
    exclude 'lib/x86_64/libgnustl_shared.so' // this line removed
}
```
